packages/indicators/src/builders/helpers.ts

```diff
@@ -40,7 +40,11 @@ export const fetchAnalytics = async (
   await Promise.all(
     Object.entries(aggregationJsonToCodes).map(async ([aggregationJson, codes]) => {
       const aggregations = JSON.parse(aggregationJson);
-      const { results } = await aggregator.fetchAnalytics(codes, fetchOptions, { aggregations });
+      const { results } = await aggregator.fetchAnalytics(
+        codes,
+        { ...fetchOptions, useDeprecatedApi: false },
```

Comment:
Will this have any unintended side affects?

Reply:
I believe no! Good decision to use this option